internal/cmd/commander.go

```diff
@@ -64,6 +66,12 @@ type FormattedCommand interface {
 	SetOutputFormat(output.Format)
 }
 
+//ConfirmedCommand should be implemented if the command should ask for user confirmation prior execution
+type ConfirmedCommand interface {
```

Comment:
This interface is not used. Probably we can check if the command implements this interface and call the method for confirmation from some more generic place. For example: in the `RunE` method of the command